### ISSUE TITLE
[REF-1030] "Global" reactive state

### ISSUE THREAD

**Issue by masenf** [reflex-dev/reflex#2052]
For values that all clients should share, but can also be updated at runtime.

For example, CMS content that may be updated in real time, but will never differ per-user.

Consider a design using a state var, GlobalVar that works like any other state var, but is actually syncronized to all clients (but stored only once on the backend).

Not sure why making [REF-245](https://linear.app/reflex-dev/issue/REF-245/global-reactive-state) public created this new issue, but we'll use it instead.

<sub>From [SyncLinear.com](https://synclinear.com) | [REF-1030](https://linear.app/reflex-dev/issue/REF-1030/global-reactive-state)</sub>
